src/views/GameView.vue

```diff
@@ -1443,8 +1441,8 @@ export default {
       'opp-score opp-score opp-score opp-score opp-score opp-score opp-score opp-score'
       'field field field field field field field field'
       'decks decks decks decks decks decks decks decks'
-      'player-score player-score player-score player-score player-score player-score player-score player-score'
-      'player-hand player-hand player-hand player-hand player-hand player-hand player-hand player-hand';
+      'score score score score score score score score'
+      'hand hand hand hand hand hand hand hand';
```

Comment:
Renaming to meet `max-len` reqs.

Reply:
ngl feels like the linter is in the way here. This is fine for now but given the way grid css works it's annoying to be constrained by the max-len here

Reply:
I’m in agreement, but we also use opp-* for the other stuff. Maybe worth turning off line lengths in the css parts of SFCs.